docs/src/main/markdown/scalapbc.md

```diff
@@ -18,8 +18,9 @@ respectively).
 ## Usage
 
 ScalaPBC is used exactly like protoc. In fact, scalapbc calls protoc.
-It ships with [multiple versions](https://github.com/os72/protoc-jar) of protoc embedded in the jar and you can pick the one you want to use by passing the desired version as the
-first command line argument:
+On the first invocation, the protoc binary will be automatically
+[fetched via Coursier](https://get-coursier.io/docs/cache.html#default-location).
```

Comment:
I was actually expecting the path to be similar to https://github.com/thesamet/sbt-protoc/blob/master/CHANGELOG.md#100-rc3, thus my PR. Is there a reason why those are not aligned?

Reply:
ScalaPB needs to touch the executable bit of what it downloads. It didn't feel right to modify caches we don't own.

Reply:
I see! Sorry I had missed the unreleased https://github.com/scalapb/ScalaPB/commit/344f8f070917f361bfab935deadaff1af1cce06b#diff-0656b67abb0945112a059b7f941e5e29903eae35eb91d28ccc9bfc986e4e29de, they are actually aligned. I'll remove the link to the Coursier cache location for now since it's irrelevant.